ci-automation/ci-config.env

```diff
@@ -53,3 +53,14 @@ QEMU_BIOS="/usr/share/qemu/bios-256k.bin"
 # Published by vms.sh as part of the qemu vendor build.
 QEMU_UEFI_BIOS="flatcar_production_qemu_uefi_efi_code.fd"
 
+
+# -- Azure --
+AZURE_HYPER_V_GENERATION="V1"
+AZURE_IMAGE_NAME="flatcar_production_azure_image.vhd"
+AZURE_LOCATION="westeurope"
```

Comment:
The location should come from the Jenkins env var, which should allocate it from a lock